### PR TITLE
Added a new field to save the token for requesting to Delete User account

### PR DESCRIPTION
**Pull request by anmode**
# Checklist:

- [x] El código sigue la [guía de
      estilo](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) de
      omegaUp.
- [x] Se corrieron todas las pruebas y pasaron.
- [ ] Si se está agregando funcionalidad nueva, se agregaron pruebas.
- [ ] Si el cambio es grande (> 200 líneas), hay que intentar partirlo en
      varios pull requests. De preferencia uno para los controladores + phpunit
      y luego otro para la interfaz.

### CODE REVIEW

File: frontend/database/00206_api_delete_account_request [hunk 1]
```diff
@@ -0,0 +1 @@
+ALTER TABLE 'Users'
```
Comment (pabo99) [inline frontend/database/00206_api_delete_account_request:1]: I think your script is incomplete. It should be:
```suggestion
-- Users
ALTER TABLE `Users`
  ADD COLUMN `deletion_token` VARCHAR( 50 ) NULL DEFAULT NULL AFTER `verification_id`;
  
```

**Review by pabo99 — COMMENTED** [review]
Just one suggestion: When you run `git push` command (from the terminal), you will be asked for fix automatically one change in `schema.sql` file. You accept this change and then push it

**Comment by anmode** [comment]
> Just one suggestion: When you run `git push` command (from the terminal), you will be asked for fix automatically one change in `schema.sql` file. You accept this change and then push it

nothing being asked when i run git push :(. Is there something wrong?

**Review by lhchavez — APPROVED** [review]
will merge after the deploy in a few hours!

**Pull request merged**